### PR TITLE
Docs 9624 pm docs edit endpoints part2 esb

### PR DESCRIPTION
**Pull request by emileemooreVT**

### CODE REVIEW

**Review by vijayang — CHANGES_REQUESTED** [review]
Please rephrase:
Endpoint Owner - The trading partner or the host that owns the endpoint. 

In HTTP/S Send endpoints:
Please change connection Idle Timeout to 30000 by default.

**Comment by emileemooreVT** [comment]
@vijayang - requested changes have been implemented. Please approve the review and I will send for Editorial.

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 1]
```diff
@@ -7,44 +7,57 @@ Create the AS2 identifiers for your organization and your trading partners in th
...
+| The trading partner or host that owns the endpoint. The value in this field is populated automatically and cannot be modified.
+| Yes
+
+| Protocol
+| The protocol selected in the `Protocol` field of the *New endpoint: Receive from partners* window.
+| Yes
+
+| Partner
+|
+| Yes
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-as2-send.adoc:17]: We need to remove this row with the setting "Partner"

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 2]
```diff
@@ -7,44 +7,57 @@ Create the AS2 identifiers for your organization and your trading partners in th
 [%header%autowidth.spread]
 |===
 |Setting |Description |Required
+
+| Owner
+| The trading partner or host that owns the endpoint. The value in this field is populated automatically and cannot be modified.
```
Comment (vijayang) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: Please remove "The value in this field is populated automatically and cannot be modified."
Depending on how the user go to the endpoint create modal, they can modify the owner (when they went to endpoint create via Message flow wizard)

**Review by vijayang — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/endpoint-as2-send.adoc (hunk 1 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-as2-send.adoc:17]: Done.

File: modules/ROOT/pages/endpoint-as2-send.adoc (hunk 2 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-as2-send.adoc]: Done. 

Also removed this sentence in the SFTP and HTTPs sent topics.

**Review by vijayang — DISMISSED** [review]

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 3]
```diff
@@ -7,44 +7,53 @@ Create the AS2 identifiers for your organization and your trading partners in th
 [%header%autowidth.spread]
 |===
 |Setting |Description |Required
+
+| Owner
+| The trading partner or host that owns the endpoint.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-send.adoc:12]: If just one entry in the Description column is a complete sentence, then this period is okay.
If all entries in the column are fragments (or could be made fragments), remove it.

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 4]
```diff
@@ -7,44 +7,53 @@ Create the AS2 identifiers for your organization and your trading partners in th
...
+| Yes
+
+| Protocol
+| The protocol selected in the `Protocol` field of the *New endpoint: Receive from partners* window.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: If this is the label of the field in the UI, then it must be formatted *Protocol*.

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 5]
```diff
@@ -7,44 +7,53 @@ Create the AS2 identifiers for your organization and your trading partners in th
...
+| Yes
+
 |Endpoint Name
-|Name used to refer to the endpoint. The name is auto populated, but you can overwrite it based on your organization’s naming conventions for consistency.
-|Yes
+| The name used to identify the endpoint. The value in the `Endpoint Name` field is generated automatically, but can be modified manually.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: Ah, I see: a sentence. So periods OK.
One option is to do something like this:

```suggestion
| The name used to identify the endpoint
+
The value in the `Endpoint Name` field is generated automatically but can be modified manually.
```

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 6]
```diff
@@ -7,44 +7,53 @@ Create the AS2 identifiers for your organization and your trading partners in th
...
 
-|Respond to email address 
-|Email address used as HTTP 'From' header when sending.
+|Respond to email address
+|The email address used as HTTP 'From' header when sending.
 |Yes
 
 |Message subject
-|Free form text for MIME Subject header
+|The user-defined field for MIME Subject header
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: Per previous, should end in a period.
Is the MIME Subject header syntax or just the name of the header?

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 7]
```diff
@@ -62,36 +71,35 @@ Allowed values include:
 |Yes
 
 |<Host> public certificate
-|This is your organization’s digital certificate (based on the RSA algorithm) for the AS2 transport protocol). 
+|Your organization’s digital certificate (based on the RSA algorithm) for the AS2 transport protocol.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: Personal pref:
```suggestion
|Your organization’s digital certificate for the AS2 transport protocol, based on the RSA algorithm.
```

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 8]
```diff
@@ -62,36 +71,35 @@ Allowed values include:
...
+|The passphrase for your private key.
 |Yes
 
 |Signature algorithm
 |The algorithm to use when setting the message integrity check value
 |Yes
 
 |MDN signature algorithm
-|Request partner to respond with a signed MDN using this algorithm for calculating MIC
+|The request partner to respond with a signed MDN using this algorithm for calculating MIC
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-send.adoc]: period

File: modules/ROOT/pages/endpoint-as2-send.adoc [hunk 9]
```diff
@@ -62,36 +71,35 @@ Allowed values include:
...
 
 |Encryption algorithm
 |The Algorithm to use when encrypting the message to be sent
 |Yes
 
 |MDN response
-|AS2 messages sent from Partner Manager always request the receiver to send a Synchronous MDN receipt back.
-Default: Sync
-|
+|The AS2 messages sent from Partner Manager always request the receiver to send a Synchronous MDN receipt back. The value in this field is set to `Sync` by default.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-as2-send.adoc:100]: Why is "synchronous" capitalized?

File: modules/ROOT/pages/endpoint-https-send.adoc [hunk 10]
```diff
@@ -12,34 +12,37 @@ Send application messages (JSON or XML) to your backend applications in inbound
...
 |===
 |Name |Description |Required
+| Owner
+| The trading partner or host that owns the endpoint. 
+| Yes
+
 |Protocol
-|Select *HTTP* or *HTTPS*. The default is *HTTPS*.
-|Yes 
+| The protocol selected in the `Protocol` field of the *New endpoint: Receive from partners* window. You can select *HTTP* or *HTTPS*. The value in this field is set to *HTTPS* by default.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-https-send.adoc]: Field name bold

File: modules/ROOT/pages/endpoint-https-send.adoc [hunk 11]
```diff
@@ -12,34 +12,37 @@ Send application messages (JSON or XML) to your backend applications in inbound
...
+|Yes
 
 |Endpoint Name
-|Name used to easily identify the function of this endpoint. Names are automatically assigned, but can be overwritten.  
-|Yes 
+|The name used to identify the endpoint. The `Endpoint Name` field is generated automatically, but can be modified manually.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-https-send.adoc]: ditto field name label

File: modules/ROOT/pages/endpoint-https-send.adoc [hunk 12]
```diff
@@ -12,34 +12,37 @@ Send application messages (JSON or XML) to your backend applications in inbound
...
 |Connection Idle Timeout (milliseconds)
-|The number of milliseconds that a connection can remain idle before it is closed. The default is `30000`.
-|Yes 
+|The connection idle timeout period in milliseconds. The value in this field is set to `30000` by default.
+|Yes
 
 |Response Timeout (milliseconds)
-|Maximum time in milliseconds that the request element blocks the execution of the flow that is waiting for the HTTP response.
-|No 
+|The maximum time in milliseconds that the request element blocks the execution of the flow that is waiting for the HTTP response.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-https-send.adoc]:
```suggestion
|The maximum time, in milliseconds, that the request element blocks the execution of the flow that is waiting for the HTTP response.
```

File: modules/ROOT/pages/endpoint-https-send.adoc [hunk 13]
```diff
@@ -49,42 +52,43 @@ Depending on how you set up your authentication policy, refer to the appropriate
 |===
 |Authentication Type |Description
 |None
-|No authentication. 
+|No authentication.
 
 |API Key
 a| * HTTP Header Name +
-Header parameter associated with the API Key. +
+The HTTP header parameter associated with the API Key. +
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-https-send.adoc]: Why is "key" init. capped in description?
If referring to the key in general, lowercase.
If referring to this as it appears in the UI, then use either "associated with *API Key*" or "associated with the *API Key* value."
GLOBAL
I know this is a pain, these subtle distinctions....

File: modules/ROOT/pages/endpoint-https-send.adoc [hunk 14]
```diff
@@ -49,42 +52,43 @@ Depending on how you set up your authentication policy, refer to the appropriate
...
-Username you set up with this authentication policy
+The username you set up with this authentication policy.
 * Password +
-Password you set up with this authentication policy 
+The password you set up with this authentication policy.
 
 |Client Credentials
 a|* Client ID +
-Client ID provided when registering your application. 
+The Client ID provided when registering your application.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-https-send.adoc]: ditto re. caps.

File: modules/ROOT/pages/endpoint-https-send.adoc [hunk 15]
```diff
@@ -49,42 +52,43 @@ Depending on how you set up your authentication policy, refer to the appropriate
...
-Add the client header. 
+Add the client header.
 * Client Secret Header +
-Add the client secret header. 
+Add the client secret header.
 
 |OAuth Token
 a|* Token URL +
-Enter a valid authorization URL. 
+Enter a valid authorization URL.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-https-send.adoc]: These three descriptions are worded as instructions. Can we reword them more like the other descriptions?

File: modules/ROOT/pages/endpoint-sftp-send.adoc [hunk 16]
```diff
@@ -1,21 +1,32 @@
...
 |Name |Description | Required
-|Endpoint Name 
-| Name used to refer to the endpoint. The name is auto-populated, but you can overwrite it based on your organization’s naming conventions for consistency.
+
+| Owner
+| The trading partner or host that owns the endpoint. 
+| Yes
+
+| Protocol
+| The protocol selected in the `Protocol` field of the *New endpoint: Receive from partners* window.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-sftp-send.adoc]: ditto prev

File: modules/ROOT/pages/endpoint-sftp-send.adoc [hunk 17]
```diff
@@ -1,21 +1,32 @@
...
+| Yes
+
+|Endpoint Name
+| The name used to identify the endpoint. The value in the `Endpoint Name` field is generated automatically, but can be modified manually.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-sftp-send.adoc]: ditto

File: modules/ROOT/pages/endpoint-sftp-send.adoc [hunk 18]
```diff
@@ -27,27 +38,27 @@ Send application messages (JSON or XML) to your backend applications in inbound
 |Yes
 
 |Username
-|Username for the SFTP Server.
+|The username for the SFTP Server.
 |Yes
 
 |Password
-|Password for the SFTP Server.
+|The password for the SFTP Server.
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-sftp-send.adoc]: Why capitalize "server" in description? ditto prev.

File: modules/ROOT/pages/endpoint-sftp-send.adoc [hunk 19]
```diff
@@ -27,27 +38,27 @@ Send application messages (JSON or XML) to your backend applications in inbound
...
 |Yes
 
 |Target Path
-a|Path to the SFTP server EDI files:
+a|The path to the SFTP server EDI files:
 
 * That are generated from outbound message flows
 * Need to be placed for your trading partners
 * Where your backend applications require you to place translated JSON and XML files generated from inbound message flows
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-sftp-send.adoc]: Does this conceptual info belong in this table?
Also note that they are not parallel.
Oops...but I was trying to stay away from the legacy text issues. However, the text that you altered is tied to this....

File: modules/ROOT/pages/endpoint-sftp-send.adoc [hunk 20]
```diff
@@ -27,27 +38,27 @@ Send application messages (JSON or XML) to your backend applications in inbound
...
 |Yes
 
 |Filename pattern
-|xref:file-name-pattern.adoc[File name pattern] as required by your trading partner or backend application
+|The xref:file-name-pattern.adoc[file name pattern] required by your trading partner or backend application
```
Comment (kbostrom) [inline modules/ROOT/pages/endpoint-sftp-send.adoc]: Note periods used above in descriptions; standardize per column per table
GLOBAL

**Review by kbostrom — DISMISSED** [review]
Completed CE; not ready for CU

File: modules/ROOT/pages/endpoint-as2-send.adoc (hunk 4 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-as2-send.adoc]: Done!

File: modules/ROOT/pages/endpoint-as2-send.adoc (hunk 6 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-as2-send.adoc]: Done - just the name of the header.

File: modules/ROOT/pages/endpoint-as2-send.adoc (hunk 8 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-as2-send.adoc]: Done.

File: modules/ROOT/pages/endpoint-as2-send.adoc (hunk 9 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-as2-send.adoc:100]: That is how it appears when it is populated in the field.

File: modules/ROOT/pages/endpoint-https-send.adoc (hunk 10 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-https-send.adoc]: Done

File: modules/ROOT/pages/endpoint-https-send.adoc (hunk 11 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-https-send.adoc]: Done.

File: modules/ROOT/pages/endpoint-https-send.adoc (hunk 13 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-https-send.adoc]: Done

File: modules/ROOT/pages/endpoint-https-send.adoc (hunk 14 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-https-send.adoc]: Done

File: modules/ROOT/pages/endpoint-https-send.adoc (hunk 15 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-https-send.adoc]: Done

File: modules/ROOT/pages/endpoint-sftp-send.adoc (hunk 16 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-sftp-send.adoc]: Done.

File: modules/ROOT/pages/endpoint-sftp-send.adoc (hunk 17 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-sftp-send.adoc]: Done.

File: modules/ROOT/pages/endpoint-sftp-send.adoc (hunk 18 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-sftp-send.adoc]: Done.

File: modules/ROOT/pages/endpoint-sftp-send.adoc (hunk 19 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-sftp-send.adoc]: Done

File: modules/ROOT/pages/endpoint-sftp-send.adoc (hunk 20 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/endpoint-sftp-send.adoc]: Done

**Review by kbostrom — DISMISSED** [review]
Reapproved

**Review by kbostrom — APPROVED** [review]
Reapproval

**Pull request merged**